command/server.go

```diff
@@ -1025,6 +1043,7 @@ func (c *ServerCommand) Run(args []string) int {
 		var err error
```

Comment:
is this declaration necessary? could you not just have `:=` on the next line? (i know this isn't part of your changes in this PR)